### PR TITLE
update slow query doc

### PR DESCRIPTION
**Pull request by crazycs520**
Signed-off-by: crazycs520 <crazycs520@gmail.com>

### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

update slow query doc to add the missing metrics about backoff

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: identify-slow-queries.md [hunk 1]
```diff
@@ -93,6 +93,8 @@ Slow Query 基础信息：
 * `Write_keys`：表示该事务向 TiKV 的 Write CF 写入 Key 的数量。
 * `Write_size`：表示事务提交时写 key 或 value 的总大小。
 * `Prewrite_region`：表示事务两阶段提交中第一阶段（prewrite 阶段）涉及的 TiKV Region 数量。每个 Region 会触发一次远程过程调用。
+* `Wait_prewrite_binlog_time`: 表示事务提交时写 binlog 的时间。
+* `Resolve_lock_time`: 表示事务提交时遇到锁后，清理或者等待锁过期的时间。
```
Comment (qiancai) [inline identify-slow-queries.md:97]:
```suggestion
* `Wait_prewrite_binlog_time`：表示事务提交时用于写 binlog 的时间。
* `Resolve_lock_time`：表示事务提交时遇到锁后，清理或者等待锁过期的时间。
```

File: identify-slow-queries.md [hunk 2]
```diff
@@ -125,13 +127,32 @@ Slow Query 基础信息：
 * `Cop_wait_p90`：cop-task 的 P90 分位等待时间。
 * `Cop_wait_max`：cop-task 的最大等待时间。
 * `Cop_wait_addr`：等待时间最长的 cop-task 所在地址。
+* `Rocksdb_delete_skipped_count`：RocksDB 读数据过程中已删除 Key 的扫描数。
+* `Rocksdb_key_skipped_count`：RocksDB 扫数据时遇到的已删除 (tombstone) Key 数量。
+* `Rocksdb_block_cache_hit_count`：RocksDB 从 Block Cache 缓存中读数据的次数。
+* `Rocksdb_block_read_count`：RocksDB 从文件系统中读数据的次数。
+* `Rocksdb_block_read_byte`：ocksDB 从文件系统中读数据的数据量。
+* `Rocksdb_block_read_time`：ocksDB 从文件系统中读数据的时间。
```
Comment (qiancai) [inline identify-slow-queries.md:135]:
```suggestion
* `Rocksdb_block_read_byte`：RocksDB 从文件系统中读数据的数据量。
* `Rocksdb_block_read_time`：RocksDB 从文件系统中读数据的时间。
```

File: identify-slow-queries.md [hunk 3]
```diff
@@ -125,13 +127,32 @@ Slow Query 基础信息：
...
+* `Rocksdb_block_read_byte`：RocksDB 从文件系统中读数据的数据量。
+* `Rocksdb_block_read_time`：RocksDB 从文件系统中读数据的时间。
 * `Cop_backoff_{backoff-type}_total_times`：因某种错误造成的 backoff 总次数。
 * `Cop_backoff_{backoff-type}_total_time`：因某种错误造成的 backoff 总时间。
 * `Cop_backoff_{backoff-type}_max_time`：因某种错误造成的最大 backoff 时间。
 * `Cop_backoff_{backoff-type}_max_addr`：因某种错误造成的最大 backoff 时间的 cop-task 地址。
 * `Cop_backoff_{backoff-type}_avg_time`：因某种错误造成的平均 backoff 时间。
 * `Cop_backoff_{backoff-type}_p90_time`：因某种错误造成的 P90 分位 backoff 时间。
 
+`Backoff_types` 一般有以下几种：
```
Comment (zhongzc) [inline identify-slow-queries.md:143]: Above I see `backoff-type`, why change its style?

Comment (crazycs520) [inline identify-slow-queries.md:143]:
```suggestion
`backoff-type` 一般有以下几种：
```

Comment (crazycs520) [reply identify-slow-queries.md]: done.

File: identify-slow-queries.md [hunk 4]
```diff
@@ -93,6 +93,8 @@ Slow Query 基础信息：
 * `Write_keys`：表示该事务向 TiKV 的 Write CF 写入 Key 的数量。
 * `Write_size`：表示事务提交时写 key 或 value 的总大小。
 * `Prewrite_region`：表示事务两阶段提交中第一阶段（prewrite 阶段）涉及的 TiKV Region 数量。每个 Region 会触发一次远程过程调用。
+* `Wait_prewrite_binlog_time`：表示事务提交时用于写 binlog 的时间。
+* `Resolve_lock_time`：表示事务提交时遇到锁后，清理或者等待锁过期的时间。
```
Comment (qiancai) [inline identify-slow-queries.md:97]: "清理" 是指什么呢

File: identify-slow-queries.md [hunk 5]
```diff
@@ -125,13 +127,32 @@ Slow Query 基础信息：
...
 * `Cop_backoff_{backoff-type}_avg_time`：因某种错误造成的平均 backoff 时间。
 * `Cop_backoff_{backoff-type}_p90_time`：因某种错误造成的 P90 分位 backoff 时间。
 
+`backoff-type` 一般有以下几种：
+
+* `tikvRPC`：给 TiKV 发送 RPC 请求失败而产生的 backoff。
+* `tiflashRPC`：给 TiFlash 发送 RPC 请求失败而产生的 backoff。
+* `pdRPC`：给 PD 发送 RPC 请求失败而产生的 backoff。
+* `txnLock`：遇到锁冲突后产生的 backoff。
+* `regionMiss`：是 Region 发生分裂或者合并后，导致 TiDB 的 region 缓存信息过期导致请求失败而产生的 backoff。
```
Comment (qiancai) [inline identify-slow-queries.md:149]:
```suggestion
* `regionMiss`：Region 发生分裂或者合并后，TiDB 的 Region 缓存信息过期导致请求失败而产生的 backoff。
```

File: identify-slow-queries.md [hunk 6]
```diff
@@ -125,13 +127,32 @@ Slow Query 基础信息：
...
+* `regionScheduling`：是 Region 还在调度中，还没有选出 Leader 导致无法处理请求而产生的 backoff。
```
Comment (qiancai) [inline identify-slow-queries.md:150]:
```suggestion
* `regionScheduling`：Region 还在调度中，尚未选出 Leader 导致无法处理请求而产生的 backoff。
```

File: identify-slow-queries.md [hunk 7]
```diff
@@ -125,13 +127,32 @@ Slow Query 基础信息：
...
+* `Rocksdb_block_read_byte`：RocksDB 从文件系统中读数据的数据量。
+* `Rocksdb_block_read_time`：RocksDB 从文件系统中读数据的时间。
 * `Cop_backoff_{backoff-type}_total_times`：因某种错误造成的 backoff 总次数。
 * `Cop_backoff_{backoff-type}_total_time`：因某种错误造成的 backoff 总时间。
 * `Cop_backoff_{backoff-type}_max_time`：因某种错误造成的最大 backoff 时间。
 * `Cop_backoff_{backoff-type}_max_addr`：因某种错误造成的最大 backoff 时间的 cop-task 地址。
 * `Cop_backoff_{backoff-type}_avg_time`：因某种错误造成的平均 backoff 时间。
 * `Cop_backoff_{backoff-type}_p90_time`：因某种错误造成的 P90 分位 backoff 时间。
 
+`backoff-type` 一般有以下几种：
```
Comment (qiancai) [inline identify-slow-queries.md:143]: backoff 的含义是什么呢

File: identify-slow-queries.md (hunk 4 above)
Comment (qiancai) [inline identify-slow-queries.md:97]: “清理”是指“清理锁”吗

**Review by qiancai — APPROVED** [review]

**Review by qiancai — APPROVED** [review]

**Pull request merged**